cmd/gateway-unsupported.go

```diff
@@ -176,6 +176,11 @@ func (a GatewayUnsupported) ListObjectsV2(ctx context.Context, bucket, prefix, c
 	return result, NotImplemented{}
 }
 
+// Walk - Not implemented stub
+func (a GatewayUnsupported) Walk(ctx context.Context, bucket, prefix string, doneCh chan struct{}) <-chan ObjectInfo {
+	return nil
```

Comment:
```suggestion
	return NotImplemented{}
```

Reply:
This is on purpose

Reply:
Why?

Reply:
Walker() doesn't need to return an error @klauspost what is the point of returning an error, there is nothing actionable to the caller. Walk() is a blind implementation which returns objects always if not the returned channel is closed.